projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -465,13 +465,14 @@ public function register_gutenberg_block() {
 		}
 
 		if ( self::is_enabled_jetpack_recurring_payments() ) {
-			$deprecated = function_exists( 'gutenberg_get_post_from_context' );
-			$uses       = $deprecated ? 'context' : 'uses_context';
 			Blocks::jetpack_register_block(
 				'jetpack/recurring-payments',
 				array(
-					'render_callback' => array( $this, 'render_button' ),
-					$uses             => array( 'isPremiumContentChild' ),
+					'render_callback'  => array( $this, 'render_button' ),
```

Comment:
Seems these changes are missing in the diff created by Fusion because this file has been already defusioned. @sdixon194 should I manually update the diff to change the `class-jetpack-memberships.php` file?

Reply:
Apologies I missed this ping! I think Jeremy and Karen have it covered it here (internal ref): pdWQjU-4v-p2#comment-83 but for visibility, the best option would be to make sure the wpcom diff gets deployed right after the wpcom weekly release goes out (which is right after Atomic goes out on Tuesdays usually). 

For testing, you can use the new script and cherry pick the wpcom diff to your sandbox. That will let you grab both changes and test them at the same time. Looks like the bot below is stuck because the build failed, but the command you would run on your sandbox should be:

`bin/jetpack-downloader test jetpack add/payment-buttons-block`

Might be easiest to `arc patch D84922`, make note of the branch it checks out, then run the above command on trunk and `git cherry-pick` the branch onto it. 

Thanks for bearing with us while we're in this weird transition period 😅